tracer/src/Datadog.Trace/IDatadogTracer.cs

```diff
@@ -19,6 +19,8 @@ internal interface IDatadogTracer
 
         ITraceSampler Sampler { get; }
 
+        GitMetadataTagsProvider GitMetadataTagsProvider { get; }
```

Comment:
Kind of surprised to see this here, given we haven't needed to expose anything else, but ok 😄

Reply:
Yeps, this is the only way I found to make it accessible not as a singleton from all the varied places we need to access it from.